src/combine.cpp

```diff
@@ -1040,7 +1040,16 @@ void do_symbol_check(chunk_t *prev, chunk_t *pc, chunk_t *next)
       tmp = chunk_get_next_ncnl(pc);
       if (chunk_is_paren_open(tmp))
       {
-         set_paren_parent(tmp, CT_DECLTYPE);
+         // decltype may be followed by a braced-init-list
+         tmp = set_paren_parent(tmp, CT_DECLTYPE);
+         if (chunk_is_comment(tmp) || chunk_is_newline(tmp))
```

Comment:
Can this ever be true? `set_paren_parent` returns `chunk_get_next_ncnl`, so I think the answer is 'no'.